.vscode/settings.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
I suggested we remove it and add it to .gitignore =P This was prompted by some leaked credentials from a previous commit (that, incidentally, were different from mine).